vagrant/base.sh

```diff
@@ -38,9 +38,31 @@ if [ -z `which javac` ]; then
 fi
 
 chmod a+rw /opt
-if [ ! -e /opt/kafka ]; then
-    ln -s /vagrant /opt/kafka
+if [ -h /opt/kafka-trunk ]; then
+    # reset symlink
+    rm /opt/kafka-trunk
 fi
+ln -s /vagrant /opt/kafka-trunk
+
+get_kafka() {
+    version=$1
+
+    kafka_dir=/opt/kafka-$version
+    url=https://s3-us-west-2.amazonaws.com/kafka-packages-$version/kafka_2.10-$version.tgz
```

Comment:
Good question. Caching only applies locally right?

In this case, I wasn't worried about the download since I added a (hopefully) convenient way to build a reusable base box, and the actual download only takes place when building the base box.